build.sbt

```diff
@@ -250,6 +251,7 @@ lazy val scioCore: Project = Project(
     "com.twitter" %% "chill" % chillVersion,
     "com.twitter" %% "chill-algebird" % chillVersion,
     "com.twitter" % "chill-protobuf" % chillVersion,
+    "com.esotericsoftware" % "kryo-shaded" % kryoVersion,
```

Comment:
Just curious, "shaded" means it contains shaded dependencies? `Kryo` itself is not shaded?

Reply:
Yes - the dependencies are shaded. Also this is the artifact chill is using. Btw, this is how the content looks like: [gist](https://gist.github.com/ravwojdyla/8ab8be9de3050ce166afd38d0ce020d8).